vars/sectionDeployToAKS.groovy

```diff
@@ -143,6 +143,14 @@ def call(params) {
                 }
               }
             }
+            if (environment == 'aat') {
+              stageWithAgent("Clear staging release - AKS $(environment", product) {
```

Comment:
AKS is unneeded, we should go through and remove from all, this is debatable whether it should have it's own stage but because of how the code is structured it's probably easier
```suggestion
              stageWithAgent("Cleanup staging release - $(environment", product) {
```